### ISSUE TITLE
Dropdown buttons should have `aria-expanded` and `aria-controls`

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#963]
**Describe the bug**
Currently, when using a SR there is nothing to indicate that the button controls the dropdown.

Adding `aria-expanded="false"` `aria-controls="<dropdown container id>"` would correct this. `aria-expanded` should be changed to `true` on click, when the dropdown is shown and reverted back to `false` when the dropdown is closed. 

Note, `aria-controls` is currently only supported in JAWS.

**To Reproduce**
1. Navigate to a dropdown button when using a SR
2. Activate the button
3. The SR is not aware the button controls the dropdown or can be expanded.

**Comment by jamesjacobs** [comment]
Resolved by #994
